### DISCUSSION TITLE
Using cinema mod breaks editor test function

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#15865] [Q&A] [answered]
Bug report:
1. Play any beatmap with the cinema mod (all gamemodes)
2. Exit out
3. Edit any beatmap and use the test function

The test function will now behave like a regular beatmap with 'cinema' turned on, which hides the gameplay elements that are necessary for testing/playing. The same behavior will remain for as long as the game is open (it does not change after playing other beatmaps with or without cinema), a restart makes this issue go away.

https://user-images.githubusercontent.com/56885706/143951704-49d0081b-9b33-4f30-9988-1aa41ded39ef.mp4

**Comment by integerrr** [comment, marked as answer]
1. you dont have to play it once to repro
2. *any* mod* will persist from song-select into editor test screen as far as I can tell
 \* with auto it also does persist into editor test screen, but the map doesnt auto play

**Reply by peppy** [reply]
Thanks, tracking via https://github.com/ppy/osu/issues/15870.
